tests/tracer/test_tracer.py

```diff
@@ -1969,3 +1969,12 @@ def test_installed_excepthook():
     assert sys.excepthook is telemetry._excepthook
     # Reset exception hooks
     telemetry.uninstall_excepthook()
+
+
+@pytest.mark.subprocess()
+def test_import_ddtrace_tracer_not_module():
+    from types import ModuleType
+
+    import ddtrace.tracer
```

Comment:
In theory nobody should be doing this to import the tracer. In general you cannot import something from a module this way (it would have to be `from ... import ...`. So if we remove `ddtrace/tracer.py` we should be fine.